plugins/signals-backend/README.md

```diff
@@ -0,0 +1,26 @@
+# signals
+
+Welcome to the signals backend plugin!
```

Comment:
Would it be possible to add some examples here to help early adopters of this plugin? Ideally it would be great to have this with the main docs but can understand we might see some iterations on this first

Reply:
Oh, shoot, I must have missed those.